src/content/docs/change-tracking/change-tracking-webhooks.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: 'Notify your team about deployments'
+tags:
+  - APM
+metaDescription: "After you set up the changes you want to monitor, you can use a webhook to notify your colleagues about the impacts of those changes."
```

Comment:
Do you have to add a redirect?

Reply:
@homelessbirds since this is a new doc that is composed of material from another doc, I don't think we need to add a redirect. We don't want to redirect users away from that other doc from APM deployment markers. I did insert a link from that doc to this new doc, so we should be OK.